content/dstu2/medication-statement.md

```diff
@@ -11,6 +11,10 @@ title: MedicationStatement | DSTU 2 API
 
 <%= terminology_table(:medication_statement, :dstu2) %>
 
+### Contained Medication Bindings
+
+<%= terminology_table(:medication, :dstu2) %>
```

Comment:
Will the contained Medication entries have the same terminology bindings for every FHIR resource they will be returned by? Since this symbol will map to a shared medication.yaml file we want to make sure it'll be accurate for each resource.